includes/html/pages/device/edit/ipmi.inc.php

```diff
@@ -3,6 +3,7 @@
 if ($_POST['editing']) {
     if (Auth::user()->hasGlobalAdmin()) {
         $ipmi_hostname = mres($_POST['ipmi_hostname']);
+        $ipmi_port = mres($_POST['ipmi_port']);
```

Comment:
mres is an empty function (it's use is legacy).
Please cast this to an integer instead.

Reply:
Great point, thank you.
I have made the change 👍